.travis.yml

```diff
@@ -40,3 +41,12 @@ script:
     if [ "$TRAVIS_TAG" == "1.0-stable" ]; then
```

Comment:
I think this section should be removed - we only want to deploy docs if the tests pass. Replace it with the line below to run the tests:
- ./build-tools/docker-docs.sh ./build-tools/make-docs.sh